<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QttMacroParaEditViewShortTimOver</class>
 <widget class="QWidget" name="QttMacroParaEditViewShortTimOver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>773</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>QttMacroParaEditViewShortTimOver</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="m_pVBoxLayout"/>
   </item>
   <item>
    <widget class="QGroupBox" name="m_boxGroupChoose">
     <property name="title">
      <string>测试项目选择</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="m_btnVolSTOver">
        <property name="text">
         <string>电压短时过量</string>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_btnCurSTOver">
        <property name="text">
         <string>电流短时过量</string>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_boxGroup">
     <property name="title">
      <string>测试方案</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="12">
       <widget class="QSttLineEdit" name="m_edtOnceInflictTime"/>
      </item>
      <item row="0" column="14">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="10" colspan="2">
       <widget class="QLabel" name="m_labOnceInflictTime">
        <property name="text">
         <string>单次施加时间</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="m_labFloadID">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="11" colspan="2">
       <widget class="QSttLineEdit" name="m_edtTotalTime"/>
      </item>
      <item row="1" column="14">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>19</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="10">
       <widget class="QLabel" name="m_labTotalTime">
        <property name="text">
         <string>试验总时长</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QSttLineEdit" name="m_edtInflictFload">
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="m_labInflictFload">
        <property name="text">
         <string>施加倍数</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="m_labInflictCount">
        <property name="text">
         <string>施加次数</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="m_labIntervalTime">
        <property name="text">
         <string>相邻施加间隔时间</string>
        </property>
       </widget>
      </item>
      <item row="1" column="13">
       <widget class="QLabel" name="m_labFloadID_3">
        <property name="text">
         <string>s</string>
        </property>
       </widget>
      </item>
      <item row="0" column="13">
       <widget class="QLabel" name="m_labFloadID_4">
        <property name="text">
         <string>s</string>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="6" colspan="4">
       <widget class="QScrollComboBox" name="m_cmbInflictCount">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2" colspan="3">
       <widget class="QSttLineEdit" name="m_edtIntervalTime"/>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="m_labFloadID_2">
        <property name="text">
         <string>s</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QScrollComboBox</class>
   <extends>QComboBox</extends>
   <header>../Module/ScrollCtrl/ScrollComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>QSttLineEdit</class>
   <extends>QLineEdit</extends>
   <header>../Controls/SttLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
